src/wms/src/test/java/org/geoserver/wms/WMSTest.java

```diff
@@ -186,6 +220,150 @@ public void doTimeElevationFilter(Object time, Object elevation, Integer... expe
                 Arrays.asList(expectedIds).containsAll(results));
     }
 
+    private static String CUSTOM_DIMENSION_NAME = WMS.DIM_ + "reference_time";
+
+    @Test
+    public void
```

Comment:
And like in the other, please use camel case and more compact names, with explanation in the javadocs

Reply:
Done.